src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js

```diff
@@ -84,14 +88,16 @@ const makeId = makeGenerator()
  * "longitude": -122.45,
  * "zoom": 15,
  * "controller": true,
+ * "pickable": true, // Enables presenting labels when hovering over Geo_Point.
```

Comment:
Thanks for clarifying. But now there seems to be a mismatch between name and comment. Are these labels selectable? If not this should not be called `pickable`.

Reply:
can we then settle on something like `showingLabels` ? By pickable i wanted to make a consistency with deck.gl approach, even though i find word "pickable" unappropriate there

Reply:
`showingLabels` sounds good to me. The deck.gl  name is just an implementation detail. The internal library used here might change, but our own API should be clear in what it does.